CHANGES.md

```diff
@@ -1,3 +1,43 @@
+## edge-19-2.4
+
+* CLI
+  * Implemented `proxy-cpu-limit` and `proxy-memory-limit` for setting the
+    proxy resources limits (`proxy-cpu` and `proxy-memory` were deprecated in
+    favor of `proxy-cpu-request` and `proxy-memory-request`) (thanks @TwinProduction!)
+  * Updated the `inject` and `uninject` subcommands to issue warnings when
+    resources lack a `Kind` property (thanks @Pothulapati!)
+  * Unhid the `install-cni` command and its flags, and tweaked their descriptions
+  * Fixed handling of kubeconfig server urls that include paths
+  * Added the `doc` command for generating docs for the CLI commands
```

Comment:
Am on the fence about this one, since `doc` is hidden, and it's only useful in the context of updating the website. How would you feel about removing this bullet point?

Reply:
Yeah agreed, something useful for us only :+1: